prov/verbs/src/verbs_cq.c

```diff
@@ -243,7 +243,7 @@ static inline int fi_ibv_poll_outstanding_cq(struct fi_ibv_ep *ep,
 		return ret;
 
 	/* Handle WR entry when user doesn't request the completion */
-	if (!wc.wr_id) {
+	if (wc.wr_id == 0xffffffff) {
```

Comment:
We should assert context != 0xffffffff in non-inject calls.

Reply:
Also, let's use a macro for this special value of wr_id.

Reply:
- assert(...) were added.
- macro `VERBS_INJECT_FLAG ((uint64_t)-1)` were defined